src/server/clients_info.rs

```diff
@@ -148,9 +148,22 @@ impl ClientInfo {
         (update_index, &mut update_info.entities)
     }
 
+    /// Sets the change limit for an entity that is replicated to this client.
+    ///
+    /// The change limit is the reference point for determining if components on an entity have changed and
+    /// need to be replicated. Component changes older than the change limit are assumed to be acked by the client.
+    pub(super) fn set_change_limit(&mut self, entity: Entity, tick: Tick) {
```

Comment:
I would prefer to name it `set_entity_tick`.

Reply:
The current "change limit" sounds odd to me...
I more comfortable with entity tick. This is how I usually referenced them. Maybe `set_change_tick`?

Reply:
Maybe `set_last_tick`?

Reply:
Is this a limit?
In my head it's the last tick since which we detect changes. Does it sounds weird?
I'm not a native speaker, so I better trust you in this.

Reply:
I thought that limit is "until", but it's actually "after".

Reply:
But when I read "limit", I think about something "until". But we detect changes not "until", but after this tick. Is this a wrong thinking?

Reply:
But why in Bevy it's called [last_change_tick](https://docs.rs/bevy/latest/bevy/ecs/component/struct.ComponentTicks.html#method.last_changed_tick) instead of `limit_change_tick`?
I suggested `last_change_tick` because it's similar to Bevy's tick...

Reply:
Okay then!